flamedisx/xenon/x1t_sr1.py

```diff
@@ -356,9 +387,11 @@ def add_extra_columns(self, d):
     @staticmethod
     def electron_detection_eff(drift_time,
                                elife,
+                               survival_probability,
                                *,
                                extraction_eff=DEFAULT_EXTRACTION_EFFICIENCY):
-        return extraction_eff * tf.exp(-drift_time / elife)
+        # electron_detection_eff comprises both the elife and the CIV effects
+        return tf.clip_by_value(extraction_eff * tf.exp(-drift_time / elife) * survival_probability ,1e-18,1) 
```

Comment:
Why 1e-18 for the clipping? sounds a bit extreme and quite arbitrary

Reply:
Because the map interpolation  returned sometimes very low values. This affected  the whole electron det eff and the algorithm didn't work.. 
Do you think e-18 is too low?